content/docs/command-reference/add.md

```diff
@@ -144,6 +144,9 @@ directory symlinks.
 - `--external` - allow `targets` that are outside of the DVC repository. See
   [Managing External Data](/doc/user-guide/managing-external-data).
 
+- `--glob` - allows adding `targets` that match the specified pattern.
```

Comment:
almost perfect! - one small suggestion to improve search- "allows adding files and directories that .... as specified by `target`"

 or `targets` can become part of the second sentence

Reply:
Sorry, I missed this comment. Updating it now.